apps/davi/src/stores/modules/common/fetchers/useGetNumberOfActiveProposals.ts

```diff
@@ -1,17 +1,13 @@
 import { useContractRead } from 'wagmi';
 import { BaseERC20Guild } from 'contracts/ts-files/BaseERC20Guild';
 import { useListenToProposalStateChanged } from '../events/useListenToProposalStateChanged';
-import { FetcherHooksInterface } from 'stores/types';
 
-type IUseGetActiveProposals = FetcherHooksInterface['useGetActiveProposals'];

```

Comment:
Hmm, but the typing is inferred from the hooks themselves. Isn't it backward to explicitly state the typings in the store and enforce it on the hooks rather than inferring it from the hooks themselves? Is there a reason we do not want the inferred typings?